autopilot-daemon/pkg/handlers/healthchecks.go

```diff
@@ -13,11 +13,12 @@ import (
 )
 
 var defaultPeriodicChecks string = "pciebw,remapped,dcgm,ping,gpupower"
+
 func PeriodicCheckTimer() {
 	klog.Info("Running a periodic check")
 	utils.HealthcheckLock.Lock()
 	defer utils.HealthcheckLock.Unlock()
-  checks, exists := os.LookupEnv("PERIODIC_CHECKS")
+	checks, exists := os.LookupEnv("PERIODIC_CHECKS")
```

Comment:
I know it's a dumb comment, but I fixed this a while ago.. I remember this because of the indentation.. Maybe a merge with main will fix